.gitignore

```diff
@@ -7,6 +7,7 @@ wgsl/index.html
 wgsl/index.pre.html
 wgsl/wgsl.lalr.txt
 wgsl/wgsl.lalr.profile
+wgsl/wgsl.recursive.bs.include.pre
```

Comment:
Thanks, I've been meaning to do this since this file always shows up as untracked on `git status`.